src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -704,6 +735,10 @@ public void canOverrideRenewalWhenItemIsNotLoanableAndNewDueDateIsSpecified() th
       item.getJson().getString("barcode")));
   }
 
+  private Matcher<ValidationError> hasRenewalWouldNotChangeDueDateMessage() {
```

Comment:
This only appears to be used once, was this extracted to improve readability?

Reply:
Yes, because of readability only